layers/+emacs/org/packages.el

```diff
@@ -50,7 +50,9 @@
     (org-sticky-header :toggle org-enable-sticky-header)
     (verb :toggle org-enable-verb-support)
     (org-roam :toggle org-enable-roam-support)
-    (valign :toggle org-enable-valign)))
+    (valign :toggle org-enable-valign)
+    (org-appear :location (recipe :fetcher github :repo "awth13/org-appear")
```

Comment:
[org-appear seems to be in MELPA](https://melpa.org/#/org-appear) -- I wonder whether it or github is preferred from where to fetch the packages?

Reply:
Good question.

Reply:
If `location` is set, github repo is preferred. So I think the one on melpa should be preferred.